test/local/account_routes.js

```diff
@@ -1195,6 +1213,77 @@ test('/account/login', function (t) {
       })
     })
   })
+
+  t.test('checks security history', function (t) {
+    var record
+    mockLog.info = sinon.spy(function(arg) {
+      if (arg.op === 'Account.history.verified') {
+        record = arg
+      }
+    })
+    var clientAddress = '127.0.0.1'
+    mockRequest.app.clientAddress = clientAddress
+
+    t.test('with a seen ip address', function (t) {
+      record = undefined
+      var securityQuery
+      mockDB.securityEvents = sinon.spy(function (arg) {
+        securityQuery = arg
+        return P.resolve([
+          {
+            name: 'account.login',
+            createdAt: Date.now(),
+            verified: true
```

Comment:
Could probably add a test case here for `verified: false` and test that the op logged is `Account.history.unverified`.